### DISCUSSION TITLE
jobrunr-spring-boot-starter Version

### DISCUSSION THREAD

**Discussion by Skillkiller** [jobrunr/jobrunr#806] [Q&A]
I use the [jobrunr-spring-boot-starter](https://central.sonatype.com/artifact/org.jobrunr/jobrunr-spring-boot-starter/5.3.3/overview) Maven package to use JobRunr integrated with Spring Boot. For some time now I get a message on the dashboard that version 6.X.X has been released. Currently I am using jobrunr-spring-boot-starter version 5.3.3. According to [Maven Central](https://central.sonatype.com/artifact/org.jobrunr/jobrunr-spring-boot-starter/5.3.3/versions) this is the latest version.

When will the jobrunr-spring-boot-starter package also be updated? The README still refers to this package.

**Comment by rdehuyss** [comment]
Hi @Skillkiller , it was already there for a long time and but the artifact id changed, so it becomes either `jobrunr-spring-boot-2-starter` or `jobrunr-spring-boot-3-starter`. It was part of JobRunr release 6 and described in the release notes.

But, thanks for pointing out the fact that the [README.md](https://github.com/jobrunr/jobrunr/blob/master/README.md) was not updated yet. That has been fixed.
